### PR TITLE
Oct 2021 for latest Windows 365 baseline version

### PR DESCRIPTION
**Pull request by kmsft**
Oct 2021 for latest Windows 365 baseline version

### CODE REVIEW

File: memdocs/intune/protect/security-baselines.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ The following security baseline instances are available for use with Intune. Use
   - [Preview: October 2019 (Edge version 77 and later)](security-baseline-settings-edge.md?pivots=edge-october-2019)
 
 - **Windows 365 Security Baseline**
-  - [Windows 365 Security Baseline version 2101](security-baseline-settings-windows-365.md)
+  - [Oct 2021](security-baseline-settings-windows-365.md)
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/security-baselines.md:95]:
```suggestion
  - [October 2021](security-baseline-settings-windows-365.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Brenduns** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**